### PR TITLE
215 dev render description logic latex

### PR DESCRIPTION
**Pull request by jb2197**
This PR adds functionality to TBox Generator to create a `.tex` file when converting TBox CSV to OWL. The class can also be called standalone.

### CODE REVIEW

**Review by sm453 — CHANGES_REQUESTED** [review]
Can you update the readme file with a few sentences that this functionality exists, and how to run it on the command line (with an existing owl file)?

File: JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md [hunk 1]
```diff
@@ -214,3 +214,21 @@ jpsBaseLibGW.importPackages(jpsBaseLib_view, "uk.ac.cam.cares.jps.base.converter
...
+    ```
+    e.g. `java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --url https://raw.githubusercontent.com/cambridge-cares/TheWorldAvatar/main/JPS_Ontology/ontology/ontodoe/OntoDoE.owl ../../JPS_Ontology/ontology/ontodoe/OntoDoE.tex`
+
+    or if the ontology is available in the local file system:
+    ```
+    java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --file <ontologyFilePath> <texFilePath>
+    ```
+    e.g. `java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --file ../../JPS_Ontology/ontology/ontodoe/OntoDoE.owl ../../JPS_Ontology/ontology/ontodoe/OntoDoE.tex`
+
+**NOTE:** A list of namespaces are defined in `TheWorldAvatar/JPS_BASE_LIB/src/main/resources/ontology.namespaces.properties` to be used as short form of IRIs, one can modify this file before running the above commands to customise the LaTeX output.
```
Comment (sm453) [inline JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md:234]: Please add the following notes:
1) If `ontology.namespaces.properties` is modified, the base lib jar file needs to be rebuilt in order for the changes to take effect.
2) If a namespace is not found in that file, the full IRI will appear in the LaTeX output. If the IRI contains a `#` character, as is often the case, this causes errors in LaTeX.
3) The LaTeX output may contain characters that are not supported, neither in `\usepackage[latin1]{inputenc}` nor `\usepackage[utf8]{inputenc}` encoding (which is the default in the CoMo preprint template). For example, `¬` and `≡` will need to be replaced with `\neg` and `\equiv`, respectively.

**Review by sm453 — CHANGES_REQUESTED** [review]

File: JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md (hunk 1 above)
Comment (jb2197) [reply JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/736/commits/6ab3d7fe603b34b0b33e9373339bd43791b98ee5

**Review by sm453 — APPROVED** [review]

File: JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/DescriptionLogicGenerator.java [hunk 2]
```diff
@@ -143,26 +143,27 @@ public static void initNamespaces() {
...
-        .replace("⊑", "$\\sqsubseteq$")
-        .replace("⊒", "$\\sqsupseteq$")
-        .replace("⊤", "$\\top$")
-        .replace("⊥", "$\\bot$")
-        .replace("∀", "$\\forall$")
-        .replace("∃", "$\\exists$")
-        .replace("→", "$\\rightarrow$")
-        .replace("⇒", "$\\Rightarrow$")
-        .replace("⇔", "$\\Leftrightarrow$")
+        .replace("⊓", "{$\\sqcap$")
```
Comment (sm453) [inline JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/DescriptionLogicGenerator.java:146]: There is a "}" missing?

**Review by sm453 — CHANGES_REQUESTED** [review]

File: JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/DescriptionLogicGenerator.java (hunk 2 above)
Comment (jb2197) [reply JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/DescriptionLogicGenerator.java]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/736/commits/c5469504e09adfdfefdfc2f36a88dafa0478f253

**Review by sm453 — APPROVED** [review]

**Pull request merged**